ex22_ch4.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: get inputs from user for three numbers
+number1 = int(input("Enter one number: "))
+number2 = int(input("Enter a second number: "))
+number3 = int(input("Enter a third number: "))
+
+#step2: set up inequalities to make sure all numbers are different. If they're not different, write a message that says that and exit the game.
```

Comment:
For this section, I originally combined all of these "elifs" on one line. However, the program didn't run correctly consistently. Also, is it preferred to combine multiple inequality statements (see below) or put them on separate lines (as shown in file)?

if number1 == number2 and number1 == number3 and number2 == number1 and number2 == number3 and number3 == number1 and number3 == number2:
    print("The numbers are not different.")
else: 
    print("Let's continue with the game.")

Reply:
Yeah, you did the overall set up correctly to keep the check that all the numbers are different from the check to see which is largest. For step 2, I'd suggest:

```
if number1 != number2 or number1 != number3 or number2 != number3:
```

This will be `True` is any of the three comparisons fail.  In your example, ` number1 == number2` is the same as `number2 == number1`, for example. Also, you'd want an `OR`. Using `and`s instead would require all the comparisons to be `True`. However, we want to know if any of them are `True`, which means we want `or`.